datastore/load-test/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:3.11-slim-bookworm
+
+SHELL ["/bin/bash", "-eux", "-o", "pipefail", "-c"]
+
+ENV DOCKER_PATH="/load-test"
+
+RUN apt-get update \
+    && apt-get -y upgrade \
+    # Cleanup
```

Comment:
Add this for ARM:
`    && apt-get install gcc libc6-dev -y --no-install-recommends \
`